usagov/vote-gov-drupal
# VOTE-1142: conditional language switcher list logic

## Jira ticket (required)
[VOTE-1142](https://bixal-projects.atlassian.net/browse/VOTE-1142)

## Description (optional)
Add logic in language switchers to show different markup based on number of translations available.

## Deployment and testing (required, if applicable)
### Pre-deploy steps
1. lando retune

### Testing steps
1. Confirm on homepage that all languages are available in the switcher dropdown
2. Confirm on http://vote-gov.lndo.site/promoting-access-to-voting that only English and Spanish languages are available with a toggle button
3. Create a new basic page with no translations and confirm no language switcher is available
